internal/data/postgres/carts/carts.go

```diff
@@ -78,20 +84,89 @@ func (c *Carts) Create(ctx context.Context, idOwner int64) (id int64, err error)
 	return dbCart.Id, nil
 }
 
-func (c *Carts) AddCartItems(ctx context.Context, items []string, cardId int64, userId int64) error {
+func (c *Carts) AddCartItems(ctx context.Context, items []string, cartId int64, userId int64) error {
 	_, err := c.conn.Exec(ctx, `
-	INSERT INTO carts_items
-		(item_name,user_id)
-	VALUES($1,$2)`,
-		items, userId)
-
+    INSERT INTO 
+        carts_items 
+        (cart_id, 
+         item_name, 
+         user_id)
+    VALUES 
+        ($1, 
+         $2, 
+         $3)
+    ON CONFLICT (
+    user_id, 
+    cart_id)
```

Comment:
не, тут как раз лучше в строчку перечислять параметры, в столбик это касалось только селектов

```suggestion
INSERT INTO (cart_id, item_name, user_id)
VALUES          (       $1,                $2,          $3)
    ON CONFLICT (user_id, cart_id)
```

Reply:
тут у гита поехали табы, суть в том что скобки и запятые друг под другом находятся